compiler-rt/lib/msan/msan_allocator.cpp

```diff
@@ -217,25 +217,52 @@ static void *MsanAllocate(BufferedStackTrace *stack, uptr size, uptr alignment,
   }
   auto *meta = reinterpret_cast<Metadata *>(allocator.GetMetaData(allocated));
   meta->requested_size = size;
+  uptr actually_allocated_size = allocator.GetActuallyAllocatedSize(allocated);
+  void* padding_start = reinterpret_cast<char*>(allocated) + size;
```

Comment:
These used only in one branch of zero, declare it there

Reply:
I think they can't be moved.

`padding_start` and `padding_size `are both used on lines 242 and 252 (separate if blocks), hence they must be defined above.

`padding_size` is defined in terms of `actually_allocated_size`, hence `actually_allocated_size` also can't be moved later.